am-lib/src/test/java/uk/gov/hmcts/reform/amlib/helpers/TestConstants.java

```diff
@@ -15,15 +18,18 @@
 
 public final class TestConstants {
 
-    public static final String ACCESS_TYPE = "user";
+    public static final AccessType ACCESSOR_TYPE = AccessType.USER;
+    public static final AccessManagementType ACCESS_MANAGEMENT_TYPE = AccessManagementType.EXPLICIT;
     public static final String SERVICE_NAME = "Service";
     public static final String RESOURCE_TYPE = "Resource Type";
     public static final String RESOURCE_NAME = "resource";
+    public static final RoleType ROLE_TYPE = RoleType.IDAM;
```

Comment:
Please avoid constants that are enumerated values. Instead of bringing value it just makes code less readable.

Reply:
👍